src/spatial/spatial_massaction_jump.jl

```diff
@@ -89,6 +89,9 @@ function SpatialMassActionJump(ma_jumps::MassActionJump{T, S, U, V}; scale_rates
                           scale_rates = scale_rates, useiszero = useiszero, nocopy = nocopy)
 end
 
+function SpatialMassActionJump(nothing)
+    SpatialMassActionJump([], [], [])
```

Comment:
`[]` does not have a concrete type. What's the use case for this?

Reply:
Maybe I should have kept it in the main PR. The use-case is to build RxRates with only constant rate jumps. Like [this](https://github.com/SciML/JumpProcesses.jl/pull/343/files#diff-e6f4dbdb2ddf3c6387947cbbda9235dc28658e3ab4a31933646662e1d1d25712R32). Do you have a suggestion for how to make an empty massaction jump better?

Reply:
Do something like we do here then to pick reasonable types:

https://github.com/SciML/JumpProcesses.jl/blob/421cb9b42a953ac0330d103933e25aa01008e1d3/src/aggregators/ssajump.jl#L125